docs/arrow.js

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react'
```

Comment:
Try to be specific about what is being imported so we don't import more than what's necessary/
```
import React, { Component } from "react";
```